ibis/backends/__init__.py

```diff
@@ -1109,11 +1109,15 @@ def _register_udfs(self, expr: ir.Expr) -> None:
         if self.supports_python_udfs:
             raise NotImplementedError(self.name)
 
+    def _table_exists(self, name: str) -> bool:
```

Comment:
Perhaps this should be call `_memtable_exists` to discourage expand the scope to include `database`/`catalog` lookups.

Reply:
You're too fast for me - I had this exact comment written in a pending review :). For consistency with the other methods it should probably be `_in_memory_table_exists` though (or rename the other ones :shrug:).

Reply:
Renamed to `_in_memory_table_exists`.